### PR TITLE
add cgroup subsystem hugetlb

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by vmarmol** [comment]
LGTM, thanks @Mashimiao!

File: cgroups/fs/hugetlb.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	return nil
+}
+
+func (s *HugetlbGroup) Remove(d *data) error {
+	return removePath(d.path("hugetlb"))
+}
+
+func (s *HugetlbGroup) GetStats(path string, stats *cgroups.Stats) error {
+	return nil
+}
```
Comment (dqminh) [inline cgroups/fs/hugetlb.go]: should we add hugetlb to the default subsystems to be joined too ?

Comment (rjnagal) [inline cgroups/fs/hugetlb.go]: I don't think its enabled everywhere by default.

**Comment by crosbymichael** [comment]
Shouldn't this be added to non-systemd code also?

**Comment by Mashimiao** [comment]
Thanks @crosbymichael ,  non-systemd added.

File: cgroups/fs/hugetlb.go (hunk 1 above)
Comment (dqminh) [inline cgroups/fs/hugetlb.go]: oops, i meant that this should be added to the list of subsystems to be applied ( which is fixed now ).

**Comment by dqminh** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**